docs/chains/doc_gen/generated-reference.mdx

```diff
@@ -85,12 +86,11 @@ an access token for downloading model weights).
 | Name | Type | Description |
 |------|------|-------------|
 | `data_dir` | *Path\|None* | The directory where the chainlet can store and access data, e.g. for downloading model weights. |
-| `user_config` | ** | User-defined configuration for the chainlet. |
-| `chainlet_to_service` | *Mapping[str,[ServiceDescriptor](#truss_chains.ServiceDescriptor* | A mapping from chainlet names to service descriptors. This is used create RPCs sessions to dependency chainlets. It contains only the chainlet services that are dependencies of the current chainlet. |
+| `chainlet_to_service` | *Mapping[str,[DeployedServiceDescriptor](#truss_chains.DeployedServiceDescriptor* | A mapping from chainlet names to service descriptors. This is used create RPCs sessions to dependency chainlets. It contains only the chainlet services that are dependencies of the current chainlet. |
```

Comment:
Are the brackets for `Mapping[str,[DeployedServiceDescriptor](#truss_chains.DeployedServiceDescriptor` supposed to be imbalanced? I noticed a few other instances of this.

Reply:
There must be a bug somewhere here: https://github.com/basetenlabs/truss/blob/main/docs/chains/doc_gen/mdx_adapter.py#L30

I couldn't track it down without spending too much time on it. I'll just fix it manually for now.